core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/runtimeModels.scala

```diff
@@ -246,7 +246,7 @@ object RuntimeImageType extends Enum[RuntimeImageType] {
   case object Jupyter extends RuntimeImageType
   case object RStudio extends RuntimeImageType
   case object Welder extends RuntimeImageType
-  case object Custom extends RuntimeImageType
+  case object VM extends RuntimeImageType
```

Comment:
I figure `RuntimeImageType.VM` is a better name than `RuntimeImageType.Custom` since it represents a VM image.

Reply:
I agree that `RuntimeImageType.VM`. I also wonder if we should distinguish between `VM` and `Docker` images for clarity. Perhaps there could be a `ImageType` as grandparent which could have children `VMImage` and `DockerImageType` and then `Jupyter`, `RStudio`, etc. could be the grandchildren.

Reply:
Yeah I never liked how this enum mixes Docker and VM images. This might be some tech debt as there are other similar models which track image types. I'd prefer to do it outside of this PR though.